src/const.ts

```diff
@@ -135,13 +135,13 @@ export const ETH_NODE_URL = ethNodeUrl
 
 export const STORAGE_KEY_LAST_PROVIDER = 'lastProvider'
 
-export const STORAGE_KEY_DISABLED_TOKENS_ADDRESSES = 'disabledTokens'
-
-export const GP_ORDER_TX_HASHES = {
-  1: 'GP_ORDER_TX_HASHES_1',
-  4: 'GP_ORDER_TX_HASHES_4',
+export const STORAGE_PENDING_ORDER_TX_HASHES = {
+  1: 'STORAGE_PENDING_ORDER_TX_HASHES_1',
+  4: 'STORAGE_PENDING_ORDER_TX_HASHES_4',
 }
 
```

Comment:
Nice job, I just have concerns on how we check if a pending transaction was mined